c/include/libsbp/ssr.h

```diff
@@ -54,6 +55,30 @@ Increased for every discontinuity in phase.
 } phase_biases_content_t;
 
 
+/** OSR correction for a particular satellite signal.
+ *
+ * Pseudorange and carrier phase corrections for a satellite being
+ * tracked.
+ */
+typedef struct __attribute__((packed)) {
+  u32 P;            /**< Pseudorange observation [2 cm] */
+  carrier_phase_t L;            /**< Carrier phase observation with typical sign convention. [cycles] */
+  u8 lock;         /**< Lock timer. This value gives an indication of the time
+for which a signal has maintained continuous phase lock.
+Whenever a signal has lost and regained lock, this
+value is reset to zero. It is encoded according to DF402 from
+the RTCM 10403.2 Amendment 2 specification.  Valid values range
+from 0 to 15 and the most significant nibble is reserved for future use.
+ */
+  u8 fix_flag;     /**< Fix type status flags.
+ */
+  sbp_gnss_signal_t sid;          /**< GNSS signal identifier (16 bit) */
+  u8 iono_var;     /**< Ionospheric correction variance [5 mm] */
```

Comment:
Hum agreed, sorry the naming is very confusing because I was think of encoding std linearly actually. Ill correct that. Let's pick u16 which should give us way enough room, and we'll refine that later